deploy/README.md

```diff
@@ -43,6 +43,7 @@ export CLOUD=azure
 export PROJECT_ID=likecoin-skynet
 export CLOUD=gcp
 
+export SSH_WHITELIST=$(curl checkip.amazonaws.com) // source cidr/ip addresses for ssh access, comma separated
```

Comment:
please move comment to another line for easier copying, and iirc bash use `#` for comments?